docs/platforms/javascript/common/index.mdx

```diff
@@ -36,6 +36,15 @@ If you prefer to follow video instructions, see [How to Install the Sentry Next.
 
 </PlatformSection>
 
+<Alert level="warning" title="Seeing deprecation warnings in your code?">
+
+We are currently preparing version 8 of the JavaScript SDKs.
+In v8, some methods and properties will be removed or renamed.
+You can refer to the [Migration docs](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#deprecations-in-7x)
+to see how you can update your code to be compatible with v8.
+
+</Alert>
```

Comment:
If you're seeing **deprecation warnings in your code**, please note that we're currently working on version 8 of the JavaScript SDKs. In v8, some methods and properties will be removed or renamed. Check out the [Migration docs](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#deprecations-in-7x)
and learn how to update your code to be compatible with v8.
```
I think that adding another alert makes this section feel quite fragmented. The user has to stop and parse a lot of information before they can get to the instructions. It would flow better if rather than making this another alert, we could add it just after the "On this page, we get you up and running with Sentry's SDK." section, so it would read: "On this page, we get you up and running with Sentry's SDK. If you're seeing **deprecations warnings in your code**, please note..."

Reply:
I updated this based on your feedback, now it is:

<img width="784" alt="image" src="https://github.com/getsentry/sentry-docs/assets/2411343/c81a2316-c482-4eff-ac60-2464d4e09958">